app/uk/gov/hmrc/cdsreimbursementclaim/models/eis/claim/enums/ReimbursementMethod.scala

```diff
@@ -34,12 +34,16 @@ object ReimbursementMethod extends EnumerationFormat[ReimbursementMethod] {
     override def toString: String = "Payable Order"
   }
 
-  final case object GeneralGuarantee extends ReimbursementMethod {
-    override def toString: String = "General Guarantee"
+  final case object Guarantee extends ReimbursementMethod {
+    override def toString: String = "Guarantee"
   }
 
-  final case object IndividualGuarantee extends ReimbursementMethod {
-    override def toString: String = "Individual Guarantee"
```

Comment:
as per the latest TPI05 schema this should be "General Guarantee" "Individual Guarantee"

```json
						"reimbursementMethod": {
							"type": "string",
							"minLength": 1,
							"maxLength": 25,
							"enum": [
								"Deferment",
								"Bank Transfer",
								"Payable Order",
								"General Guarantee",
								"Individual Guarantee"
							]
						},
```